src/common_utils/language_utils/sentence_filter_utils.py

```diff
@@ -19,33 +20,40 @@ def initialize_database():
         list.append({'text': r})
     db.add_many_new_docs_to_collection('polish_stop_words', list)
 
+
 def from_txt_file_to_list(path):
     file = open(path, "r")
     lines = list(map(lambda x: x.rstrip(), list(file.readlines())))
     return lines
 
+
 def filter_word_form(word_form, morphologic_tag):
     return len(morphologic_tag.intersection(word_class_name.get(word_form))) > 0
 
+
 def delete_additional_info_after_colon(word):
     index = word.find(':')
     if index == -1:
         return word
     return word[:index]
```

Comment:
I think colon should be passed as a parameter to this function, it will just make it more flexible

Reply:
it is method from Andrzej's branch so @Andrzej97 consider it also